### DISCUSSION TITLE
Multi AZ LB not found as multi az

### DISCUSSION THREAD

**Discussion by nomaddevops** [Azure/azqr#241] [Q&A] [answered]
Hello there :)

Just a quick question regarding the test of the load balancers.
According to this documentation https://learn.microsoft.com/en-us/azure/reliability/reliability-load-balancer?tabs=graph, Standard SKU Load Balancer supports availability zones and zone resiliency, while the Basic SKU doesn't.

From my understanding this means, that if we have a Standard LB it should be by default using all the available zone, right ?

However in azqr my lb is detected as not in HA. Is it a false positive or my understanding of the docs isn't right ?

**Comment by cmendible** [comment, marked as answer]
The LB must not use the Basic SKU and the IP's used in the frontend or public ip configuration must be AZ enabled : https://learn.microsoft.com/en-us/azure/reliability/reliability-load-balancer?tabs=graph#availability.

Check the IPs an let me know.

**Comment by nomaddevops** [comment]
Look like the IP is misconfigured, az is set no zone
